emails/templates/emails/first_time_user.html

```diff
@@ -0,0 +1,411 @@
+{% comment %}
+  Note that Django only loads strings from `app.ftl`, `brands.ftl`, and `phones.ftl`
+  so make sure all email strings are included in those files.
+{% endcomment %}
+{% load ftl %}
+{% load email_extras %}
+{% withftl bundle='privaterelay.ftl_bundles.main' language=language %}
+
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional //EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="{{ request.LANGUAGE_CODE }}" dir="ltr"></html>
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="x-apple-disable-message-reformatting">  
+  <title>Firefox Relay</title>
+
+  <style type="text/css">
+    @font-face {
+      font-family: 'inter';
+      font-style: normal;
+      src: local('Inter Regular'), local('Inter-Regular'), url(https://relay.firefox.com/fonts/Inter/Inter-Regular.4232a675a077.woff2?v=3.18) format('woff');
+    } 
+    @font-face {
+      font-family: 'inter medium';
+      font-style: bold; 
+      src: local('Inter Medium'), local('Inter-Medium'), url(https://relay.firefox.com/fonts/Inter/Inter-Medium.woff2?v=3.18) format('woff');
+    }
+    @font-face {
+      font-family: 'metropolis medium';
+      font-style: bold; 
+      src: local('Metropolis Medium'), local('Metropolis-Medium'), url(https://relay.firefox.com/fonts/Metropolis/Metropolis-Medium.woff2) format('woff');
+    }
+    @font-face {
+      font-family: 'metropolis bold';
+      font-style: bold; 
+      src: local('Metropolis Bold'), local('Metropolis-Bold'), url(https://relay.firefox.com/fonts/Metropolis/Metropolis-Bold.woff2) format('woff');
+    }
+    body {
+      font-family: 'inter', Arial, sans-serif;
+      color: #3D3D3D;
+    }
+    h1 {
+        font-family: 'metropolis bold';
+    }
+    a {
+      color: #0060DF;
+      text-decoration: underline;
+      text-underline-offset: 4px;
+    }
+    .button {
+        box-sizing: border-box;
+        padding: 15px 25px;
+        display: inline-block;
+        font-family: 'metropolis medium';
+        text-decoration: none;
+        -webkit-text-size-adjust: none;
+        text-align: center;
+        color: #FFFFFF; 
+        background-color: #0060df; 
+        border-radius: 4px;
+        -webkit-border-radius: 4px;
+        -moz-border-radius: 4px; 
+        width:auto; 
+        max-width:100%; 
+        overflow-wrap: break-word; 
+        word-break: break-word; 
+        word-wrap:break-word;
+        font-size: 16px;
+    }
+    p {
+      margin: 0;
+    } 
+    a[x-apple-data-detectors='true'] {
+      color: inherit ;
+      text-decoration: none ;
+    } 
+    table,
+    td {
+      color: #000000;
+    }  
+    #get-relay-premium-section {
+      display: block; 
+      width: 98%;
```

Comment:
This is an interesting magic number? Could possibly do with a comment detailing where those 2% went? (And I guess most importantly, does it work across screen sizes?)

Reply:
Updated and added comment